client/src/Components/Mentors.js

```diff
@@ -5,59 +5,118 @@ import {
 	Route,
 	Link,
 } from "react-router-dom";
-import Questions from "../mockData/Questions.json";
+// import Questions from "../mockData/Questions.json";
 
 import "../App.css";
 import "../grid.css";
 
-export default function Mentors() {
-	const [fetchedQuestions,setfetchedQuestions]=useState(Questions);
+export default function Mentors(props) {
+	// const [fetchedQuestions,setfetchedQuestions]=useState(props.questions);
 	const [newQuizzQuestions, setNewQuizzQuestions]=useState([]);
+	const [refreshQuestions, setRefreshQuestions]=useState(false);
+
 	const [newQuizz,setNewQuizz]=useState(	{
-		id: 1,
-		name: "javaScrypt week 1",
-		publiShingDate: "2020-08-04",
-		questions: [],
+
+		name: "",
+		publiShingDate: "",
+		questions_id: [],
 	});
+
+
+
 	useEffect(() => {
-		if(newQuizz.questions!==null){
 
 
+
+		const makeQuestions=()=>{
 			let newQuizQuestions = [];
-			newQuizQuestions = newQuizz.questions.map((selId)=>{
+			newQuizQuestions = newQuizz.questions_id.map((selId)=>{
 
-				let found = (Questions.find((question)=>question.id==selId));
+				let found = (props.questions.find((question)=>question._id==selId));
 				newQuizQuestions.push(found);
-
-
 				setNewQuizzQuestions(newQuizQuestions);
-
 			}
 			);
-		}
-	},[newQuizz]);
+
+		};
+		makeQuestions();
+	},[newQuizz.questions_id]);
+
+
+
 	const addQuestion =(e)=>{
+		console.log(e.target.value);
 		setNewQuizz({ ...newQuizz,
-			questions:[...newQuizz.questions,e.target.value],
+			questions_id:[...newQuizz.questions_id,e.target.value],
 		},
 		);
 
 	};
+	const submitNewQ =()=>{
```

Comment:
Can you rename this? We have both `question` and `quiz` which both start with Q, so it's not obvious what's being submitted here.